.kitchen.yml

```diff
@@ -128,6 +128,9 @@ platforms:
       name: vagrant
       gui: true
     provisioner:
+      # On Osx 'brew install unix2dos' or Centos 'yum install tofrodos'
+      cache_commands:
+        - find %{sandbox_path} -type f | xargs unix2dos
```

Comment:
I would actually rather just have a test that just does like... touch %{sandbox_path}/file and then a test that checks if that file is there, so it can be tested on linux, since those are the only tests that run on prs right now.